settings/Validation/FormValidator.js

```diff
@@ -15,6 +15,10 @@ const defaultValidators = {
     validate: (value) => isNumber(value) || !isEmpty(value),
     message: <FormattedMessage id="ui-circulation.settings.validate.fillIn" />,
   },
+  isNotEmptySelect: {
+    validate: (value) => isNumber(value) || !isEmpty(value),
```

Comment:
Please use validators composition for this case

Reply:
Done.